plugins/linear-genome-view/src/index.ts

```diff
@@ -142,6 +146,51 @@ export default class LinearGenomeViewPlugin extends Plugin {
         },
       })
     }
+
+    this.initFromLocString(pluginManager)
+  }
+
+  async initFromLocString(pluginManager: PluginManager) {
```

Comment:
maybe call this initFromQueryString instead

Reply:
fixed :)